bridge/github/export_mutation.go

```diff
@@ -0,0 +1,74 @@
+package github
+
+import "github.com/shurcooL/githubv4"
+
+type createIssueMutation struct {
+	CreateIssue struct {
+		Issue struct {
+			ID  string `graphql:"id"`
+			URL string `graphql:"url"`
+		}
+	} `graphql:"createIssue(input:$input)"`
+}
+
+type updateIssueMutation struct {
+	UpdateIssue struct {
+		Issue struct {
+			ID  string `graphql:"id"`
+			URL string `graphql:"url"`
+		}
+	} `graphql:"updateIssue(input:$input)"`
+}
+
+type addCommentToIssueMutation struct {
+	AddComment struct {
+		CommentEdge struct {
+			Node struct {
+				ID  string `graphql:"id"`
+				URL string `graphql:"url"`
+			}
+		}
+	} `graphql:"addComment(input:$input)"`
+}
+
+type updateIssueCommentMutation struct {
+	UpdateIssueComment struct {
+		IssueComment struct {
+			ID  string `graphql:"id"`
+			URL string `graphql:"url"`
+		} `graphql:"issueComment"`
+	} `graphql:"updateIssueComment(input:$input)"`
+}
+
+type removeLabelsFromLabelableMutation struct {
+	AddLabels struct {
+		Labelable struct {
+			Typename string `graphql:"__typename"`
+		}
+	} `graphql:"removeLabelsFromLabelable(input:$input)"`
+}
+
+type addLabelsToLabelableMutation struct {
+	RemoveLabels struct {
+		Labelable struct {
+			Typename string `graphql:"__typename"`
+		}
+	} `graphql:"addLabelsToLabelable(input:$input)"`
+}
+
+type createLabelMutation struct {
+	CreateLabel struct {
+		Label struct {
+			ID string `graphql:"id"`
+		} `graphql:"label"`
+	} `graphql:"createLabel(input: $input)"`
+}
+
+type createLabelInput struct {
```

Comment:
You could PR that upstream

Reply:
Will do ! The[ input.go](https://github.com/shurcooL/githubv4/blob/master/input.go) file is generated from  https://github.com/shurcooL/githubv4/blob/master/gen.go which use `https://api.github.com/graphql` to get all objects descriptions. I'll open an issue first to see how we can add new objects.